tests/regression/roles/sonic_pki/defaults/main.yml

```diff
@@ -0,0 +1,100 @@
+---
+ansible_connection: httpapi
+module_name: tacacs_pki
+
+preparations_tests:
+  delete_pki:
+    - name: delete_all
+      description: Delete PKI
+      state: deleted
+      input:
+tests:
+  - name: test_case_01
+    description: Add new PKI configuration
+    state: merged
+    input:
+      security_profiles:
+        - profile_name: mysp
+        - profile_name: myspb
+          ocsp_responder_list:
+            - http://example.com/ocsp
+      trust_stores:
+        - name: myts
+          ca_name: CA2
+
+  - name: test_case_02
+    description: Update specific params of security-profile
```

Comment:
Please expand this test case or add another one to test the following list expansion cases:

- Adding one or more ca_names to an existing trust_store ca_name list.
- Adding one or more URLs to an existing security profile ocsp_responder list.
- Adding another trust store and/or security profile.